tasks/resolve_reference.json

```diff
@@ -17,6 +17,13 @@
     "credentials": {
       "type": "Optional[String]"
     },
+    "aws_access_key_id": {
+      "type": "Optional[String]"
```

Comment:
I think i see why this is not "sensitive" but your test case makes me hesetant https://github.com/puppetlabs/puppetlabs-aws_inventory/pull/17/files#diff-c7639b8a584e34dc59a315a1f3b5d23034c8a64db7415137a76c525bae92f89bR106 given you call it "secret" there?

Reply:
That's the other field. `aws_access_key_id` is not sensitive, `aws_secret_access_key` is sensitive.